CONTRIBUTING.md

```diff
@@ -1,15 +0,0 @@
-# Contributing
-Hello fellow contributor! This document will provide an overview of the process for contributing to the CBC casper codebase! Please note that these guidelines may be changed in the future. 
-
-# Getting Started
-To get started with the codebase, see the download and installation instructions in the readme. 
-
-# Making a Pull Request
-Once you have implemented your proposed changes, you need to create a pull request. Before making a PR, make sure there are no merge conflicts with your branch and that all tests pass.
-
-TODO: Add notes about licensing and rights. 
```

Comment:
this is sort of implied by having the license, no?

Reply:
yes